orc8r/protos/BUILD.bazel

```diff
@@ -72,10 +107,19 @@ proto_library(
 proto_library(
     name = "mconfigs_proto",
     srcs = ["mconfig/mconfigs.proto"],
-    strip_import_prefix = "mconfig/",
```

Comment:
does this break the existing build? I'm forgetting why I put the strip import

Reply:
As far as I can see  this rule `mconfigs_proto` is only used by the rule `mconfigs_python_proto` in the same BUILD file. In this case the path is required to generate the python file in correct folder.